From c3d677ced12399716cb1559d230bf12c294fda9b Mon Sep 17 00:00:00 2001
From: Gokul Sriram Palanisamy <quic_gokulsri@quicinc.com>
Date: Thu, 24 Oct 2024 15:53:23 +0530
Subject: [PATCH] bus: mhi: host: update SDX65 data channels

Increase number of elements in HW data channel to keep sufficient buffers available at higher throughputs.

Change-Id: Ic7a1c531e80509641ed8d5d739b9aa7f39f21115
Signed-off-by: Gokul Sriram Palanisamy <quic_gokulsri@quicinc.com>
---
 drivers/bus/mhi/host/pci_generic.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/bus/mhi/host/pci_generic.c b/drivers/bus/mhi/host/pci_generic.c
index 51e45f6c50b5..5b67fb784b75 100644
--- a/drivers/bus/mhi/host/pci_generic.c
+++ b/drivers/bus/mhi/host/pci_generic.c
@@ -270,8 +270,8 @@ static const struct mhi_channel_config modem_qcom_v1_mhi_channels[] = {
 	MHI_CHANNEL_CONFIG_DL_FP(35, "FIREHOSE", 32, 0),
 	MHI_CHANNEL_CONFIG_UL(46, "IP_SW0", 64, 1),
 	MHI_CHANNEL_CONFIG_DL(47, "IP_SW0", 64, 3),
-	MHI_CHANNEL_CONFIG_HW_UL(100, "IP_HW0", 128, 4),
-	MHI_CHANNEL_CONFIG_HW_DL(101, "IP_HW0", 128, 5),
+	MHI_CHANNEL_CONFIG_HW_UL(100, "IP_HW0", 512, 4),
+	MHI_CHANNEL_CONFIG_HW_DL(101, "IP_HW0", 512, 5),
 	MHI_CHANNEL_CONFIG_HW_DL(103, "IP_HW_QDSS", 128, 10),
 	MHI_CHANNEL_CONFIG_UL(105, "RMNET_CTL", 128, 6),
 	MHI_CHANNEL_CONFIG_DL(106, "RMNET_CTL", 128, 7),
-- 
2.34.1

